SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -11,38 +12,43 @@ Buffer {
 	*initClass { serverCaches = IdentityDictionary.new }
 
 	// doesn't send
-	*new { arg server, numFrames, numChannels, bufnum;
+	*new { arg server, numFrames, numChannels, sampleRate, bufnum;
```

Comment:
New arguments must always come last! Otherwise this can break existing code.

Reply:
that is an interesting bias. I will respect it even if I disagree - user interface logic first is my motto, but you're the boss :)

Reply:
This is a misunderstanding: of course, for new classes, we'd always take extra care for the best order of arguments. But we almost never change an existing interface later on. We try to keep deprecation to an absolute minimum and by all means try not to break existing code in new versions.

Reply:
I understand your perspective. I will implement accordingly.